### ISSUE TITLE
♿Pagination AVT1 - August 2018♿

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1362]
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
Each form control must have associated label
<img width="477" alt="screen shot 2018-10-30 at 1 41 43 am" src="https://user-images.githubusercontent.com/40970507/47700589-0dc7ed00-dbe5-11e8-9dab-422f4be284ac.png">

## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset
